sql/src/test/java/org/apache/druid/sql/calcite/schema/SystemSchemaTest.java

```diff
@@ -469,7 +485,7 @@ public Object get(String name)
         100L,
         2L, //partition_num
         1L, //num_replicas
-        3L, //numRows
+        2L, //numRows
```

Comment:
this change is because I created another `ROWS3` and `index3` above, such that now `(segment3, index3)`  is added to `walker`, to make the test code more intuitive.

Reply:
Thanks.